compiler/modules/front.syntax/token.bal

```diff
@@ -487,9 +487,90 @@ public readonly class SourceFile {
         return unpackPosition(pos);
     }
 
-    public function lineContent(Position pos) returns string {
-        int lineNum = self.lineColumn(pos)[0];
-        return scanLineToString(self.scannedLine(lineNum));
+    // range is expected to be the start of a token
```

Comment:
range will be at the start of a fragment but not necessarily the start of the token

Reply:
comment fixed (logic already assumed it is the beginning of a fragment)